### DISCUSSION TITLE
User is too new! Please check with support.

### DISCUSSION THREAD

**Discussion by vasnetsov** [cirruslabs/cirrus-ci-docs#976] [Q&A]
Trying to start build, but there is a bot protection or what. How long do I need to wait to be non-new user?

**Comment by fkorotkov** [comment]
Lately there is a huge uptick in attempts to mine crypto currencies using free CI/CD services including Cirrus CI. This is the first level of defense: a user should be **at least two days old** to create builds for their personal repositories. Note that **there is an exception for PR builds for repositories that already had Cirrus CI configured** so you should be good to open PRs right away.

**Comment by smithpb2250** [comment]
I came here with the same question. Can you please add this delay period information to the FAQ so it is easier to find?
